### PR TITLE
Reduce noise to avoid false loop feel

### PR DESCRIPTION
**Pull request by jotadrilo**
By using a large set of plugins, it feels like the tool is iterating forever. However, it just takes time to download and process that amount of plugins.

### CODE REVIEW

File: pkg/plugins/meta/meta.go [hunk 1]
```diff
@@ -31,44 +31,43 @@ func Print(pm *api.PluginMetadata) {
...
-	return nil, nil
+	return nil
 }
 
 func worker(id int, jobs <-chan *metadataRequest, results chan<- error) {
 	for mr := range jobs {
-		log.Printf("#%2d> fetching %s metadata...\n", id, mr.Plugin.Identifier())
-		_, err := FetchMetadata(mr.Plugin, mr.WorkingDir)
-		results <- err
+		results <- FetchMetadata(mr.Plugin, mr.WorkingDir)
```
Comment (migmartri) [inline pkg/plugins/meta/meta.go]: are these results or errors?

Comment (migmartri) [reply pkg/plugins/meta/meta.go]: I see, it's a channel of errors. The name can be confusing though.

File: pkg/plugins/meta/meta.go [hunk 2]
```diff
@@ -31,44 +31,43 @@ func Print(pm *api.PluginMetadata) {
 }
 
 // FetchMetadata will fetch the metadata for the requested plugin
-func FetchMetadata(p *api.Plugin, workingDir string) (*api.PluginMetadata, error) {
+func FetchMetadata(p *api.Plugin, workingDir string) error {
```
Comment (migmartri) [inline pkg/plugins/meta/meta.go]: does this function need to be exported? Just in case.

Comment (jotadrilo) [reply pkg/plugins/meta/meta.go]: it does not need to be exported so far, but I also exported `FetchPlugin` so I wanted to standardize those running a similar task.

File: pkg/plugins/meta/meta.go (hunk 1 above)
Comment (jotadrilo) [reply pkg/plugins/meta/meta.go]: I didn't want to conflict with the `errors` package and I think it is confusing until you see the parameter type. If this concern raises again I will consider using a different name for sure.

**Pull request merged**